centre-api/src/centre_api/services/access_requests.py

```diff
@@ -66,27 +67,45 @@ def process_access_request(cls, access_request_id, status):
             raise PermissionError(
                 f"User does not have permission to update access requests for app '{app_name}'."
             )
-
-        return cls._do_update_access_request(access_request, status)
+        access_request_response = None
+        app = access_request.app
+        auth_user_response = AuthApiService.get_user_by_id(access_request.user_auth_guid)
+        with session_scope() as session:
+            access_request_response = cls._do_update_access_request(access_request, status, session)
+            _queue_access_denied_email(session, app, auth_user_response)
```

Comment:
I know it's weird that we have "approve" request in one service and "deny" in another. We can perhaps revisit to improve that. But this "process_access_request" function is general (although yes currently it's only called with deny). Can we perhaps just queue an email and in that queue email function there is a condition on the status and it calls deny if status is deny?